### PR TITLE
GAWB-1697: allow users to save incomplete metadata

### PR DESCRIPTION
**Pull request by skwalker**
With https://github.com/broadinstitute/firecloud-orchestration/pull/412
Disables buttons so you can't republish / publish with incomplete metadata

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by skwalker** [comment]
not sure I've tested the last commit enough but kinda running out of time

File: src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs [hunk 1]
```diff
@@ -207,7 +212,8 @@
                               last-page (> page-num (-> library-schema :wizard count))]
                         [comps/Button {:text (if published? "Republish" "Submit")
                                        :onClick #(react/call :submit this)
```
Comment (ahaessly) [inline src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs]: Should this be "Save" instead of "Submit"?

**Comment by ahaessly** [comment]
if a workspace has some attributes set and I go to the summary page, it looks like I can publish the dataset. Only if I actually edit some value does the publish button get disabled.
don't worry about fixing - i'll pick it up next week 😸

**Comment by ahaessly** [comment]
this has been replace by the branch ah-gawb-1697

**Pull request closed without merging**